src/GridFieldOrderableRows.php

```diff
@@ -755,6 +756,23 @@ protected function getManyManyInspector($list)
         return $inspector;
     }
 
+
+    /**
+     * Depending on the list inspector and the list itself (ManyMany vs ManyManyThrough), the method to obtain
+     * the foreign key may be different.
+     *
+     * @param $inspector
+     * @return string
+     */
+    protected function getManyManyInspectorForeignKeyMethod($inspector)
+    {
+        if (($inspector instanceof ManyManyThroughQueryManipulator) && (method_exists($inspector, 'getForeignIDKey'))) {
+            return 'getForeignIDKey';
```

Comment:
Any reason we should return a string here instead of calling the method and returning its result?